conf/application.conf

```diff
@@ -1,4 +1,4 @@
-# Copyright 2023 HM Revenue & Customs
+# Copyright 2004 HM Revenue & Customs
 #
```

Comment:
In this file, please remove the following, thanks:

- [JsonErrorHandler](https://github.com/hmrc/tax-history/blob/DDCE-5261/conf/application.conf#L20-L22) comes in from [bootstrap-play](https://github.com/hmrc/bootstrap-play/blob/main/bootstrap-backend-play-30/src/main/resources/backend.conf#L42)
- the comments [here](https://github.com/hmrc/tax-history/blob/DDCE-5261/conf/application.conf#L30-L43) and [here](https://github.com/hmrc/tax-history/blob/DDCE-5261/conf/application.conf#L101-L106)
- [here](https://github.com/hmrc/tax-history/blob/DDCE-5261/conf/application.conf#L106-L118) and configure [logback.xml](https://github.com/hmrc/tax-history/blob/DDCE-5261/conf/logback.xml) accordingly. 
- [metrics](https://github.com/hmrc/tax-history/blob/DDCE-5261/conf/application.conf#L119-L124) and [auditing](https://github.com/hmrc/tax-history/blob/DDCE-5261/conf/application.conf#L130-L133) configurations
- [datastream](https://github.com/hmrc/tax-history/blob/DDCE-5261/conf/application.conf#L155-L159)